opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -1878,43 +1892,83 @@ const completeSpecialFilterKeys = async (context, user, inputFilters) => {
     const filter = filters[index];
     const { key } = filter;
     const arrayKeys = Array.isArray(key) ? key : [key];
-
-    if (arrayKeys.includes(TYPE_FILTER)) {
-      // in case we want to filter by entity_type
-      // we need to add parent_types checking (in case the given value in type is an abstract type)
-      const { newFilter, newFilterGroup } = adaptFilterToEntityTypeFilterKey(filter);
-      if (newFilter) {
-        finalFilters.push(newFilter);
+    if (arrayKeys.some((fiterKey) => complexConversionFilterKeys.includes(fiterKey))) {
+      if (arrayKeys.length > 1) {
+        throw Error(`A filter with these multiple keys is not supported: ${arrayKeys}`);
+      }
```

Comment:
I'm a bit confused about the necessity of this condition. Above, we verify whether we have one or more complexFilterKeys. If there are more than one, we throw an error.